pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -310,3 +317,38 @@ func (p *PolicyData) EnsureRBACPolicy() genericapiserver.PostStartHookFunc {
 func (p RESTStorageProvider) GroupName() string {
 	return rbac.GroupName
 }
+
+// ClusterRolesToAggregate maps from previous clusterrole name to the new clusterrole name
+var ClusterRolesToAggregate = map[string]string{
```

Comment:
agree the map of role name to aggregated role name should come from bootstrap policy, not be inlined here